tests/oval_details/test_oval_details.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# OpenSCAP test suite
+# Tests OVAL details in OpenSCAP reports
+# Last change: 29.1.2015
```

Comment:
Why list Last change in the file? This will inevitably be the wrong value when somebody changes this file and doesn't notice this is there. If people want this info they can just do `git log $FILE`.